server/src/main/java/io/spine/server/entity/RecordBasedRepository.java

```diff
@@ -313,7 +329,7 @@ protected E findOrCreate(I id) {
         checkNotNull(filters);
         checkNotNull(fieldMask);
 
-        final EntityQuery<I> entityQuery = EntityQueries.from(filters, getEntityClass());
+        final EntityQuery<I> entityQuery = EntityQueries.from(filters, columnCache());
```

Comment:
Similar to the JDBC library, let's hide the entity column cache from the public API. `EntityQueries.from(...)` is a good example of external API.